#Header header
#    uint32 seq
#    time stamp
#    string frame_id
uint8 TYPE_DEFAULT 		=0
uint8 TYPE_AUTO_MAN			=1
uint8 TYPE_DISABLE_TELE_ASS	=2
uint8 TYPE_CONFIRM			=3
uint8 TYPE_CANCEL 			=4 
uint8 TYPE_RETURN_HOME  =5
uint8 TYPE_FOCUS_CAM		=6
uint8 TYPE_MENUE			  =7

uint8 TYPE_FRONT_CAM		=8
uint8 TYPE_BACK_CAM			=9
uint8 TYPE_MANI_CAM			=10
uint8 TYPE_THERMO_CAM		=11
uint8 TYPE_KINECT_CAM		=12
uint8 TYPE_FIX_CAM			=17

uint8 TYPE_LED_ON       = 18
uint8 TYPE_HOOK_TOGGLE_ACTION  = 19
uint8 TYPE_HOOK_HOME    = 20

uint8 status

